frontend/src/components/VMediaInfo/VMediaDetails.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <section class="flex flex-col gap-y-6">
+    <header class="flex flex-row items-center justify-between">
+      <h2 class="heading-6 md:heading-5">
+        {{ $t(`mediaDetails.${media.frontendMediaType}Info`) }}
+      </h2>
+      <VContentReportPopover :media="media" />
+    </header>
+    <div class="flex flex-col items-start gap-6 md:flex-row">
+      <slot name="thumbnail" />
+
+      <div
+        class="flex w-full flex-grow flex-col"
+        :class="
+          media.frontendMediaType === 'audio' ? 'gap-4 lg:gap-6' : 'gap-6'
+        "
+      >
+        <p v-if="media.description">{{ media.description }}</p>
+        <VMediaTags :tags="media.tags" />
+        <VMetadata v-if="metadata" :metadata="metadata" :media="media" />
+      </div>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue"
+
+import type { AudioDetail, ImageDetail, Metadata } from "~/types/media"
+
+import { useI18n } from "~/composables/use-i18n"
+import { getMediaMetadata } from "~/utils/metadata"
+
+import VContentReportPopover from "~/components/VContentReport/VContentReportPopover.vue"
+import VMetadata from "~/components/VMediaInfo/VMetadata.vue"
+import VMediaTags from "~/components/VMediaInfo/VMediaTags.vue"
+
+export default defineComponent({
+  components: {
+    VMediaTags,
+    VMetadata,
+    VContentReportPopover,
+  },
+  props: {
+    media: {
+      type: Object as PropType<AudioDetail | ImageDetail>,
+      required: true,
+    },
+    imageWidth: {
```

Comment:
`image` here makes me think that this might be specific to image media, although I know that the album art is likely used here for audio records. Does maybe `previewWidth` or `mediaWidth` make more sense, to make this as generic as possible? Or should we stick with the `image` prefix?

Reply:
It's actually the property that's specific for the images. All of the audio thumbnails are square. Some images, on the other hand, don't have the values for width, height and image type in the API result. So, we send a head request to the provider, and update these values before displaying them. This is why the values are set separately here.